CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.10.0
```

Comment:
Would probably make sense to make this PR `v1.9.0` as it looks good to go, whereas that PR has comments waiting to be fixed.